presto-main/src/main/java/com/facebook/presto/execution/SqlQueryExecution.java

```diff
@@ -616,12 +671,20 @@ private void planDistribution(PlanRoot plan)
         }
     }
 
+    /**
+     * Try to cancel the execution of the query.
+     * TODO : Add more details on how cancellation request is propogated to tasks, connectors etc
```

Comment:
Please fix this TODO

Reply:
I added this TODO, because I'm not very clear on how cancellation works! I'm aware that there can be cases where tasks continue executing even after the query is cancelled. If you have more details, I can add those.